src/utilities.ts

```diff
@@ -50,6 +50,12 @@ export function getMinorFullName(acronym: string): string {
   return minor ? minor.name : '';
 }
 
+export function getGradFullName(acronym: string): string {
```

Comment:
Haha yeah I thin I wrote this function too. Will resolve after this is merged!

Reply:
Sounds good!